react-app/src/App.js

```diff
@@ -0,0 +1,34 @@
+import './App.css';
+import axios from "axios";
+import Greeting from './components/Greeting';
+import { useEffect, useState } from 'react';
+const API_URL="http://localhost:3000/api/v1/greetings";
+
+function getAPIData(){
+  return axios.get(API_URL).then((response)=> response.data)
+}
+
+function App() {
+  const [greetings, setGreetings]=useState([]);
+useEffect(()=>{
+  let mounted=true;
+  getAPIData().then((items)=>{
```

Comment:
- The whole react-redux part should be implemented inside ` app/javascript` folder.